chess-AI/train.py

```diff
@@ -22,19 +22,23 @@ class MctsTrain:
         mcts: References the MCTS class
         policy_converter: References the PlayNetworkPolicyConverter class
     """
-    def __init__(self, mcts_simulations, exploration, lr):
-        # Set the number of Monte Carlo Simulations
-        self.mcts_simulations = mcts_simulations
-        self.mcts = Mcts(exploration)
-
+    def __init__(self, lr, move_approximator, val_approximator=None):
         self.policy_converter = PlayNetworkPolicyConverter()
 
         # Learning rate
         self.lr = lr
 
-    def training_game(self, nnet):
-        # Stores final probability distribution from MCTS
-        mcts_probs = []
+        # board -> move_list, move_probabilities, move_to_take
+        self.move_approximator = move_approximator
+
+        # board -> state_value
+        # if None, values will be based on the game outcome
+        self.val_approximator = val_approximator
+
+    def training_game(self):
+        # Stores probability distributions and values from the approximators
```

Comment:
This comment should 1) be a docstring and 2) explain the return values more clearly